tests/stub/notification_filters/scripts/run_notification_filters.script

```diff
@@ -0,0 +1,11 @@
+!: BOLT 5.1
+
+A: HELLO {"{}": "*"} {"{}": "*"}
+?: RESET
+C: RUN "RETURN 1 as n" {} {"[mode]": "*", "[db]": "*", "notifications{}": #FILTERS#}
+S: SUCCESS {"fields": ["n"]}
+C: PULL {"n": 1000}
```

Comment:
Non-pipelining drivers will do `DISCARD` here and in similar places. However, we can update these in a separate PR if it makes it easier.

Reply:
I'll sort it. no issue.